### ISSUE TITLE
Opening quotation mark with ellipsis problem

### ISSUE THREAD

**Issue by michaelfranzl** [gettalong/kramdown#253]
This

    Kramdown::Document.new('"abc def"').to_latex

correctly puts opening and closing quotation marks:

    ``abc def''

Whereas having an ellipsis at the beginning of the quote

    Kramdown::Document.new('"...abc def"').to_latex

gives two closing quotation marks

     ''\\ldots{}abc def''

Behavior is the same for HTML and LaTeX output. It's not a big issue, but having an ellipsis at the beginning of a quotation is quite common in typesetting.

**Comment by gettalong** [comment]
The smart quote feature in kramdown is certainly not complete and getting this right in general is a really hard problem. If I were to create kramdown now I would probably leave the smart quotes out completely.

However, I think that in this case it would a) be easy to add and b) pose no compatibility problems - I will have a look.

**Comment by gettalong** [comment]
I have added some code to allow the requested behavior. However, I'm not entirely sure if the change has any side effects but I don't think so. The test suite still passes but that is not a 100% verification.

Tl;dr: Will be in the next release!
